cloudos_cli/__main__.py

```diff
@@ -8,11 +8,14 @@
 import json
 import time
 import sys
-import os
-import urllib3
 from ._version import __version__
 from cloudos_cli.configure.configure import ConfigurationProfile
 from cloudos_cli.datasets import Datasets
+import re
+from cloudos_cli.utils.resources import ssl_selector, format_bytes
+from rich.console import Console
+from rich.table import Table
+from rich.style import Style
 
```

Comment:
```suggestion
from rich.style import Style


```
Expected 2 blank lines